### PR TITLE
Sherwani/tidbits views

### PR DESCRIPTION
**Pull request by HussainKhanSherwani**
# Scroll bar was hidden and message was added for user to take action before it was unresponsive

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [x] I have carefully reviewed my code and believe it is final from my side.
- [x] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [x] I have tested the new feature or updates in both light and dark themes.
- [x] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [x] I have tested my changes on mobile screen sizes.
- [x] I have tested my changes on tablet screen sizes.
- [x] I have tested my changes on laptop screen sizes.

### Functional Testing
- [x] I have thoroughly tested the functionality of my changes.
- [x] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [x] Explain the task/bug you are solving
- [x] Show the changes in the UI
- [x] Show the changes in the codebase
- [x] Show the changes you tested to make sure it works
- [x] Show the changes in various screen sizes
videourl:https://www.loom.com/share/3211a3c958bc4cebb1ca17253e961e08?sid=3d3ff04b-7375-47d3-b339-d2c420a93022

### CODE REVIEW

File: academy-ui/src/components/bytes/View/ByteStepperItem/ByteStepperItemWithProgressBar.module.scss [hunk 1]
```diff
@@ -17,3 +17,11 @@
...
   box-shadow: 0 2px 4px var(--primary-color);
 }
+.hideScrollbar {
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
+}
+
+.hideScrollbar::-webkit-scrollbar {
+  display: none; /* Safari and Chrome */
+}
```
Comment (MSamiTariq) [inline academy-ui/src/components/bytes/View/ByteStepperItem/ByteStepperItemWithProgressBar.module.scss:27]: @RobinNagpal what would suggest, should we remove the scrollbar or keep it?

Comment (RobinNagpal) [reply academy-ui/src/components/bytes/View/ByteStepperItem/ByteStepperItemWithProgressBar.module.scss:27]: @HussainKhanSherwani  does the content scroll now when there is a image and quite a bit of content on the step? What happens in that case?
// cc @MSamiTariq

Comment (HussainKhanSherwani) [reply academy-ui/src/components/bytes/View/ByteStepperItem/ByteStepperItemWithProgressBar.module.scss:27]: yes it scrolls the content but no scroll bar is visible it is hidden

Comment (RobinNagpal) [reply academy-ui/src/components/bytes/View/ByteStepperItem/ByteStepperItemWithProgressBar.module.scss:27]: @HussainKhanSherwani  if you feel these changes are good and you have tested it, I will merge it. 

Thanks for verifying the details.  
// cc @MSamiTariq

Comment (HussainKhanSherwani) [reply academy-ui/src/components/bytes/View/ByteStepperItem/ByteStepperItemWithProgressBar.module.scss:27]: sure
